Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/continuouspipe/symfony-php7.1-nginx:stable
+FROM quay.io/continuouspipe/symfony-php7.1-nginx:latest
```

Comment:
stable doesn't have trusted ip support. Need to test a release for new stable. I think it works.